pkgs/development/tools/rust/cargo-lambda/default.nix

```diff
@@ -1,33 +1,47 @@
 { lib
+, cacert
 , rustPlatform
 , fetchFromGitHub
 , makeWrapper
+, pkg-config
+, openssl
 , stdenv
 , Security
-, cargo-watch
 , zig
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "cargo-lambda";
-  version = "0.13.0";
+  version = "0.18.1";
 
   src = fetchFromGitHub {
     owner = pname;
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-JwrTwCn4qXcjXjy4cPYM3WCblJS0J5Xxnc6WESMwCpU=";
+    sha256 = "sha256-un+GQflxhMHCMH5UEeUVsYx59ryn7MR4ApooeOuhccc=";
   };
 
-  cargoSha256 = "sha256-p3q5S6IFQQgNp/MHGSUE1DVLFyMLWDTv/dxrUACKSWo=";
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+    outputHashes = {
+      "cargo-test-macro-0.1.0" = "sha256-XvTKAbP/r1BthpEM84CYZ2yfJczxqzscGkN4JXLgvfA=";
+    };
+  };
 
-  nativeBuildInputs = [ makeWrapper ];
+  nativeBuildInputs = [ cacert makeWrapper pkg-config ];
```

Comment:
`cacert` doesn't seem to be required to build, and isn't referenced by the output, so I think it should be in `nativeCheckInputs`.